### DISCUSSION TITLE
Tablet issues Linux Mint XFCE

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#23754] [Q&A] [answered]
![osu_2023](https://github.com/ppy/osu/assets/40531871/3cc2efea-05b3-47dd-9838-09ed0a6beade)

Got this issue in 2019 but the issue is still there today so I figure maybe talking about it

iirc installing OTD fixes the issue but I don't know if this is intended considering OTD is installed with lazer on win.
I figured maybe because I use an AppImage there is no setup. But I thought maybe osu could get the package on first startup, maybe its not voluntarily failing

**Comment by bdach** [comment, marked as answer]
See https://opentabletdriver.net/Wiki/FAQ/Linux#fail-device-streams

We can't fix this ourselves without `sudo`, so the best we can probably do is an error message.

**Reply by AuracleTech** [reply]
I see, thanks
